docs/QUICKSTART.md

```diff
@@ -161,6 +161,18 @@ Eraser can be used to [automatically](#automatically-cleaning-images) or [manual
     docker.io/library/alpine:3.7.3                                                                             application/vnd.docker.distribution.manifest.list.v2+json sha256:8421d9a84432575381bfabd248f1eb56f3aa21d9d7cd2511583c68c9b7511d10 2.0 MiB   linux/386,linux/amd64,linux/arm/v6,linux/arm64/v8,linux/ppc64le,linux/s390x  io.cri-containerd.image=managed
     docker.io/library/alpine@sha256:8421d9a84432575381bfabd248f1eb56f3aa21d9d7cd2511583c68c9b7511d10           application/vnd.docker.distribution.manifest.list.v2+json sha256:8421d9a84432575381bfabd248f1eb56f3aa21d9d7cd2511583c68c9b7511d10 2.0 MiB   linux/386,linux/amd64,linux/arm/v6,linux/arm64/v8,linux/ppc64le,linux/s390x  io.cri-containerd.image=managed
     ```
+## Excluding images 
+Eraser can exclude registries (docker.io/library/*) and also specific images with a tag (azurecr.io/myImage:tag) or id (sha256:80f31da1ac7b312ba29d65080fd...) from its removal process.
```

Comment:
```suggestion
Eraser can exclude registries (example, `docker.io/*`) and also specific images with a tag (example, `docker.io/library/ubuntu:18.04`) or digest (example, `sha256:80f31da1ac7b312ba29d65080fd...`) from its removal process.
```

Reply:
can we exclude all tags `docker.io/library/ubuntu:*`?

Reply:
it cant right now but I can add a check for that